src/ResourceManager/Security/Commands.Security/help/AzureRM.Security.md

```diff
@@ -0,0 +1,76 @@
+---
+Module Name: AzureRM.Security
+Module Guid: {{ Update Module Guid }}
+Download Help Link: {{ Update Download Link }}
+Help Version: {{ Update Help Version }}
+Locale: {{ Update Locale }}
+---
+
+# AzureRM.Security Module
+## Description
+{{Manually Enter Description Here}}
```

Comment:
@chlahav please make sure a quick description about the module is added here

Reply:
Done